src/plugins/auto/src/plugin.cpp

```diff
@@ -248,7 +250,11 @@ std::vector<DeviceInformation> Plugin::parse_meta_devices(const std::string& pri
                     full_device_name = get_core()->get_property(device_name_with_id, ov::device::full_name);
                 } catch (ov::Exception&) {
                     LOG_DEBUG_TAG("get full device name failed for ", device_name_with_id.c_str());
+                OPENVINO_SUPPRESS_DEPRECATED_START
+                } catch (InferenceEngine::Exception&) {
```

Comment:
Why do we need old exception here?

Reply:
ilya, I just tried to align to ov::exception in plugin calls, but seems introduce several case failures in different components,  so shall we merge this first? I can try in another PR to fix the exceptions below:
https://github.com/openvinotoolkit/openvino/blob/master/src/inference/src/dev/plugin.cpp#L18

Reply:
Yes let's do it separatelly.